docker-app/qfieldcloud/subscription/models.py

```diff
@@ -804,14 +804,16 @@ def create_subscription(
             active_since (Optional[datetime]): active since for the subscription.
 
         Returns:
-            Tuple[Subscription, Subscription]: the created trial subscription if the given plan was a trial and the regular subscription.
+            tuple[AbstractSubscription | None, AbstractSubscription]: the created trial subscription if the given plan was a trial and the regular subscription.
 
         TODO Python 3.11 the actual return type is Self
         """
         if active_since:
             # remove microseconds as there will be slight shift with the remote system data
             active_since = active_since.replace(microsecond=0)
 
+        assert active_since
+
```

Comment:
@why-not-try-calmer @faebebin 

This assert is actually breaking the system, as we have situations where we DO allow no active since to be passed. You can clearly see that from the typing of the function argument.

Reply:
Fixed.

Reply:
thanks for fixing :+1: